### PR TITLE
DBZ 2567 Prepare revised SMT docs for downstream

### PR DESCRIPTION
**Pull request by roldanbob**
To address a security issues with the filter, and content-based routing SMTs, were removed from the core product, and the documentation was revised (DBZ-2549) to describe how to obtain and set up the SMTs.   
This issue is for updates related to preparing the revised documentation for use downstream. The updates include:

- Heading annotation comments to support splitting/modularizing the files for downstream use.
- Language edits to comply with standard product doc use. 
- Minor restructuring/shuffling of content to improve flow.

This change needs to be backported to 1.2.

### CODE REVIEW

File: documentation/modules/ROOT/pages/configuration/content-based-routing.adoc [hunk 1]
```diff
@@ -33,27 +33,31 @@ The {prodname} content-based routing SMT is a Technology Preview feature. Techno
...
+Instead, it is provided in a separate artifact, `debezium-scripting-{debezium-version}.tar.gz`.
+To use the content-based routing SMT with a {prodname} connector plug-in, you must explicitly add the SMT artifact to your Kafka Connect environment.
 
-Note that this allows anyone who can set up connectors in your Kafka Connect instance to execute scripting expressions.
-You should therefore only add this SMT after securing your Kafka Connect instance and its configuration interface appropriately.
-
-{prodname} itself doesn't come with any implementations of the JSR 223 API.
-To use an expression language, you must download the JSR 223 script engine implementation for the language, and add to your {prodname} connector plug-in directories, along any other JAR files used by the language implementation.
-Eg. for Groovy 3, you can download its JSR 223 implementation from https://groovy-lang.org/. For GraalVM JavaScript, this is is available at https://github.com/graalvm/graaljs.
+IMPORTANT: After the filter SMT is present in a Kafka Connect instance, any user who is allowed to add a connector to the instance can run scripting expressions.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/configuration/content-based-routing.adoc:59]:
```suggestion
IMPORTANT: After the routing SMT is present in a Kafka Connect instance, any user who is allowed to add a connector to the instance can run scripting expressions.
```

File: documentation/modules/ROOT/pages/configuration/content-based-routing.adoc [hunk 2]
```diff
@@ -33,27 +33,31 @@ The {prodname} content-based routing SMT is a Technology Preview feature. Techno
...
+To ensure that scripting expressions can be run only by authorized users, be sure to secure the Kafka Connect instance and its configuration interface before you add the filter SMT.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/configuration/content-based-routing.adoc:60]:
```suggestion
To ensure that scripting expressions can be run only by authorized users, be sure to secure the Kafka Connect instance and its configuration interface before you add the routing SMT.
```

**Review by gunnarmorling — COMMENTED** [review]
LGTM, thanks, @roldanbob. Two small suggestions for c&p glitches.

**Comment by gunnarmorling** [comment]
Rebased and applied to master/1.3. Will do so for 1.2 too shortly.

Thanks, @roldanbob!

File: documentation/modules/ROOT/pages/configuration/content-based-routing.adoc (hunk 2 above)
Comment (roldanbob) [reply documentation/modules/ROOT/pages/configuration/content-based-routing.adoc]: Oh! Bad copy/paste. Sorry and thanks for catching it!

**Comment by gunnarmorling** [comment]
Applied to 1.2, too.

**Pull request merged**